scm/driver/bitbucket/webhook.go

```diff
@@ -598,6 +607,137 @@ type (
 		Repository  prCommentHookRepo        `json:"repository"`
 		Actor       prCommentHookUser        `json:"actor"`
 	}
+
+	pipelineHook struct {
```

Comment:
just check if there's any convention to where to place structs and where to place functional methods in single file

Reply:
Using the same convention as we have earlier